### PR TITLE
Update googshift and olcs

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

**Comment by sbrunner** [comment]
To be rebased ...

**Comment by gberaudo** [comment]
done

File: buildtools/webpack.commons.js [hunk 1]
```diff
@@ -134,6 +134,7 @@ const config = {
       'moment': 'moment/src/moment.js',
       'ol/ol.css': 'openlayers/css/ol.css',
       'ol': 'openlayers/src/ol',
+      'goog': 'olcs/goog/',
```
Comment (sbrunner) [inline buildtools/webpack.commons.js]: Why this?

Comment (gberaudo) [reply buildtools/webpack.commons.js]: You are right, this is a remaining from when I was on an old branch.

Comment (sbrunner) [reply buildtools/webpack.commons.js]: ok :-)

**Pull request merged**